documentation/src/docs/metrics.md

```diff
@@ -7,491 +7,466 @@ You may obtain a copy of the License at
+    controller.stream.truncate_failed - with tags {"scope", $scope, "stream", $stream}
+
+  ```
+
+- Controller Stream retention frequency ([Counter](https://micrometer.io/docs/concepts#_counters)) and truncated size ([Gauge](https://micrometer.io/docs/concepts#_gauges)) Metrics:
+  ```
+    controller.retention.frequency - with tags {"scope", $scope, "stream", $stream}
+
+    controller.retention.truncated_size - with tags {"scope", $scope, "stream", $stream}
+
+  ```
+
+- Controller Stream Segment operations ([Counters](https://micrometer.io/docs/concepts#_counters)) and open/timed out Transactions on a Stream ([Gauge](https://micrometer.io/docs/concepts#_gauges)) Metrics  - all with tags {"scope", $scope, "stream", $stream}:
+  ```
+    controller.transactions.opened
+    controller.transactions.timedout
+    controller.segments.count.$scope
+    controller.segment.splits.$scope
+    controller.segment.merges.$scope
+  ```
+
+- Controller Transaction operation latency Metrics:
+  ```
+    controller.transactions.created_latency_ms
+    controller.transactions.committed_latency_ms
+    controller.transactions.aborted_latency_ms
+  ```
+
+- Controller Transaction operation counter Metrics:
+    ```
+    controller.transactions.created_global
+    controller.transactions.created - with tags {"scope", $scope, "stream", $stream}
+    controller.transactions.create_failed_global
+    controller.transactions.create_failed - with tags {"scope", $scope, "stream", $stream}
+    controller.transactions.committed_global
+    controller.transactions.committed - with tags {"scope", $scope, "stream", $stream}
+    controller.transactions.commit_failed_global
+    controller.transactions.commit_failed - with tags {"scope", $scope, "stream", $stream}
+    controller.transactions.commit_failed - with tags {"scope", $scope, "stream", $stream,  "transaction", $txnId}
+    controller.transactions.aborted_global
+    controller.transactions.aborted - with tags {"scope", $scope, "stream", $stream}
+    controller.transactions.abort_failed_global
+    controller.transactions.abort_failed - with tags {"scope", $scope, "stream", $stream}
+    controller.transactions.abort_failed - with tags {"scope", $scope, "stream", $stream,  "transaction", $txnId}
+  ```
+
+- Controller hosts available ([Gauge](https://micrometer.io/docs/concepts#_gauges)) and host failure ([Counter](https://micrometer.io/docs/concepts#_counters)) Metrics:
+  ```
+    controller.hosts.count
+    controller.hosts.failures_global
+    controller.hosts.failures.$host - with tags {"host", $host}
+
+  ```
+
+- Controller Container count per host ([Gauge](https://micrometer.io/docs/concepts#_gauges)) and failover ([Counter](https://micrometer.io/docs/concepts#_counters)) Metrics:
+  ```
+    controller.hosts.container_count
+    controller.container.failovers_global
+    controller.container.failovers.$containerId - with tags {"container", $containerId}
```

Comment:
Remove $containerId from the metric name.

Reply:
Accepted suggestion